docs/core/compatibility/cryptography/6.0/cfb-mode-feedback-size-exception.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Breaking change: CreateEncryptor methods throw exception for incorrect feedback size"
+description: Learn about the breaking change in .NET 6 where the `CreateEncryptor` and `CreateDecryptor` methods for `AesCng` and `TripleDESCng` throw an exception for an incorrect feedback size when used with a persisted key for CFB mode.
+ms.date: 10/01/2021
+---
+# CreateEncryptor methods throw exception for incorrect feedback size
+
+The `CreateEncryptor` and `CreateDecryptor` methods for <xref:System.Security.Cryptography.AesCng> and <xref:System.Security.Cryptography.TripleDESCng> now throw a <xref:System.Security.Cryptography.CryptographicException> when the object instance is being used with a CNG persisted (or named) key for Cipher Feedback (CFB) mode, with a feedback size other than eight (CFB8).
+
+## Previous behavior
+
+Previously, these classes allowed CFB128 (AesCng) or CFB64 (TripleDesCng) to be selected. However, if the key was a persisted key, then the computation was always done as if CFB8 was selected.
+
+## New behavior
+
+The `CreateEncryptor` and `CreateDecryptor` methods throw a <xref:System.Security.Cryptography.CryptographicException> when both of the following conditions are met:
+
+- CFB128 or CFB64 mode is selected (that is, <xref:System.Security.Cryptography.AesCryptoServiceProvider.FeedbackSize?displayProperty=nameWithType> is set to 128 or <xref:System.Security.Cryptography.TripleDESCryptoServiceProvider.FeedbackSize?displayProperty=nameWithType> is set to 64).
```

Comment:
@bartonjs I'm not sure I used the right properties here. Can you check?

Reply:
Neither AesCng nor TripleDESCng override that property, so they're both `<xref:System.Security.Cryptography.SymmetricAlgorithm.FeedbackSize?displayProperty=nameWithType>`, I think.  (If "inherited properties" auto xref down, then it could be `<xref:System.Security.Cryptography.AesCng.FeedbackSize?displayProperty=nameWithType>`)